packages/docs/src/lib/liveCode/components/LiveCodeEditor/LiveCodeEditor.module.css

```diff
@@ -58,6 +58,11 @@
   padding: var(--size-px--l);
 }
 
+.center {
```

Comment:
Bisher haben wir den Inhalt der Code Beispiele in der Umsetzung nirgends zentriert, ich glaube bei UX ist es aber immer zentriert, wenn dann sollten wir es auch überall machen, damit es einheitlich ist

Reply:
ja würd ich sagen